docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/memberlist.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Migrating from Consul to Memberlist KV store for hash rings"
+menuTitle: "Migrating from Consul to Memberlist"
+description: "Learn how to migrate from using Consul as KV store for hash rings to using memberlist without any downtime."
+weight: 30
+---
+
+# Migrating from Consul to Memberlist as KV store for hash rings without downtime
+
+Mimir Jsonnet uses `memberlist` as KV store for hash rings, unless disabled by using
```

Comment:
```suggestion
Mimir Jsonnet uses `memberlist` as KV store for hash rings since Mimir 2.2.0.

Memberlist can be disabled by using the following configuration:
```